pages/documentation/migration-V3.md

```diff
@@ -178,7 +176,7 @@ Add this load path to your compiler settings, or update any old paths if your co
 </h4>
 <div id="m-a6" class="usa-accordion__content site-prose">
   <ol>
-    <li>How do you know you're using USWDS Gulp? Search for a line like <code>const uswds = require("./node_modules/uswds-gulp/config/uswds");</code> or <code>const uswds = "node_modules/uswds/dist"</code> in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.</li>
```

Comment:
`Gulp` is not capitalized on the gulp website https://gulpjs.com/

Reply:
Agreed